doc/subsystems/bluetooth/qualification.rst

```diff
@@ -15,3 +15,28 @@ the following documents:
    l2cap-pics.rst
    sm-pics.rst
    rfcomm-pics.rst
+
+Qualification Listings
+**********************
+
+The Zephyr BLE stack has obtained qualification listings for both the Host
+and the Controller.
+See the tables below for a list of qualification listings
+
+Host qualifications
+-------------------
+
+=============== ======================================================= ===================
+Zephyr version  Link                                                    Qualifying Company
+=============== ======================================================= ===================
+1.13            https://launchstudio.bluetooth.com/ListingDetails/70189 Nordic
+=============== ======================================================= ===================
+
+Controller qualifications
+-------------------------
+
+=============== ======================================================= =================== ===================
+Zephyr version  Link                                                    Qualifying Company  Compatible Hardware
+=============== ======================================================= =================== ===================
+1.9 to 1.13     https://launchstudio.bluetooth.com/ListingDetails/25166 Nordic              nRF52x
```

Comment:
The launchstudio page (that I can see without a login/account) only mentions Zephyr version 1.9x

Reply:
again strange, no idea why that is the case, i will check with someone

Reply:
Here's what I see...

![image](https://user-images.githubusercontent.com/11063618/47925886-6d8bf580-de7c-11e8-8125-c413ed22e15d.png)